### PR TITLE
Admin: fix links to automattic.com and jetpack.com using http

### LINKED ISSUE #5048: Admin Page: Link to automattic.com in footer is http
I realize this is low priority, but I figured that we should still make a not of it.

The "An Automattic Airline" link that is displayed in the footer is non-https.

screen shot 2016-08-31 at 1 40 55 pm

### PR DESCRIPTION
**Pull request by eliorivero**
Fixes #5048

#### Changes proposed in this Pull Request:
- change `http` to `https` in links to automattic.com and jetpack.com 

#### Testing instructions:
- go to Jetpack admin and check that the following links are correct:

- [ ] Need help? link at the top bar
- [ ] Automattic logo at the footer, where it says, **An Automattic airline**
- [ ] 4.3-rc1 link at the footer
- [ ] Privacy link at the footer

### CODE REVIEW

File: _inc/client/components/footer/index.jsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ export const Footer = React.createClass( {
 		return (
 			<div className={ classes }>
 				<div className="jp-footer__a8c-attr-container">
-					<a href="http://automattic.com" target="_blank">
+					<a href="https://automattic.com" target="_blank">
 					<svg className="jp-footer__a8c-attr" x="0" y="0" viewBox="0 0 935 38.2" enable-background="new 0 0 935 38.2">
```
Comment (ebinnion) [inline _inc/client/components/footer/index.jsx]: Unrelated to this changeset, but the indentation is a bit off. Down a few lines we also do `</svg></a>` on the same line. Just making a note.

**Comment by ebinnion** [comment]
I haven't built/tested this PR yet. But, the changeset LGTM. 👍

**Comment by rcoll** [comment]
Indentation fixed. Built & tested all good! :ship:

**Comment by zinigor** [comment]
Tested, looks good, thanks!

**Pull request merged**